website/pages/docs/developers/publishing-a-plugin-to-the-hub.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Publishing a Plugin to the Hub
+description: Learn how to publish a plugin to the Hub and have it listed in https://hub.cloudquery.io/
+---
+
+# Publishing a Plugin to the Hub
+
+With the announcement of [CloudQuery Hub](blog/announcing-cloudquery-new-hub), we are excited to see the community contribute plugins to the Hub. This guide will walk you through the process of publishing a plugin to the Hub.
+
+:::callout
+Publishing to the CloudQuery Hub is only supported for Go plugins at the moment. We are working on adding support for other languages.
+:::
+
+## Prerequisites
+
+- You have created a [CloudQuery Cloud](https://cloud.cloudquery.io/) account and completed the onboarding process to create a team
+- You have created the plugin you'd like to publish on https://cloud.cloudquery.io/ under the relevant team
+- You have the [CloudQuery CLI](https://cloudquery.io/docs/cli/installation) installed (version >= `v3.27.1`)
+- The plugin you'd like to publish is written in Go, and uses an SDK version >= `v4.17.1`
+- The plugin you'd like to publish is initialized using the plugin's name, team and kind. See example [here](https://github.com/cloudquery/cloudquery/blob/b7ef6f6ed8948272a429f35614fa28559397227a/plugins/source/test/resources/plugin/plugin.go#L15)
+- You are authenticated to [CloudQuery Cloud](https://cloud.cloudquery.io/) using the `cloudquery login` command
+
+## Publishing a Plugin
+
+1. In the root directory of your plugin repository run `git tag v1.0.0` to tag the version you're about to publish (replace `v1.0.0` with the version you'd like to publish)
+2. Run `git push origin v1.0.0` to push the tag
```

Comment:
The first two steps aren't strictly necessary; I think you'd only do this if you are running this as part of a CI process

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/15104/commits/1d548ff5ea43052d2da98f526e19c54b545d079b